### PR TITLE
Clean up example SCSS file

### PR DESCRIPTION
**Pull request by isti115**
Remove some duplicated code and adjust whitespace

### CODE REVIEW

File: examples/eww-bar/eww.scss [hunk 1]
```diff
@@ -31,24 +32,11 @@
   margin-left: 10px;
   margin-right: 20px;
 }
-.metric scale trough highlight {
-  all: unset;
-  background-color: #D35D6E;
-  color: #000000;
-  border-radius: 10px;
-}
```
Comment (elkowar) [inline examples/eww-bar/eww.scss:39]: If you remove the highlight styles here, this _does_ break the UI

Comment (w-lfchen) [reply examples/eww-bar/eww.scss:39]: i regret to inform you that it doesn't, both are duplicated.
as of ee937b90ec0b19d545dc4f415156cd375e0b0fa2, [lines 19-33](https://github.com/elkowar/eww/blob/ee937b90ec0b19d545dc4f415156cd375e0b0fa2/examples/eww-bar/eww.scss#L19-L33) are the exact same as [34-48](https://github.com/elkowar/eww/blob/ee937b90ec0b19d545dc4f415156cd375e0b0fa2/examples/eww-bar/eww.scss#L34-L48).

**Review by w-lfchen — APPROVED** [review]
lgtm

**Comment by isti115** [comment]
Thanks for the validation, @w-lfchen! :+1:
(Seeing a notification for a PR sent almost two years ago feels a bit surreal. :smiley:)

**Comment by w-lfchen** [comment]
you're welcome!

**Pull request merged**